<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>finUiGraphConfigDlg</class>
 <widget class="QDialog" name="finUiGraphConfigDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Graphics Configuration</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>390</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>541</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Panel</string>
   </property>
   <widget class="QLabel" name="lblBgColorTitle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Background Color:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="finUiColorLabel" name="lblBgColor">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblPanelSizeTitle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Panel Size:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblPanelWidthTitle">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Width</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblPanelWidthUnit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>21</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>px</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="spbPanelWidth">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="spbPanelHeight">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>50</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblPanelHeightUnit">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>50</y>
      <width>21</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>px</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblPanelHeightTitle">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Height</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="ckbPanelSizeRatio">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Keep the width-height ratio.</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblOriginPointTitle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Origin Point:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="spbOriginPtX">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="spbOriginPtY">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>110</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblOriginPtXUnit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>110</y>
      <width>21</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>px</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblOriginPtYUnit">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>110</y>
      <width>21</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>px</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblOriginPtXTitle">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>X:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblOriginPtYTitle">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>110</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Y:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="ckbOriginPtRatio">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>140</y>
      <width>311</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Keep the respective position to the panel size.</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>541</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Axis</string>
   </property>
   <widget class="QLabel" name="lblPanelSizeTitle_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Axis Unit Size:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="minimum">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="lblOriginPtXUnit_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>21</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>px</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>finUiColorLabel</class>
   <extends>QLabel</extends>
   <header>finUiColorLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>finUiGraphConfigDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>finUiGraphConfigDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
